src/implementations/twig/components/news-ticker/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@ecl/twig-component-news-ticker",
+  "author": "European Commission",
+  "license": "EUPL-1.2",
+  "version": "3.0.2",
+  "description": "ECL News ticker",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@ecl/twig-component-button": "^3.0.2",
```

Comment:
would you update this component and its dependencies to the current ECL version, 3.0.3?